scripts/setup_irods.py

```diff
@@ -63,6 +63,12 @@ def wrap_if_necessary():
 from irods.password_obfuscation import maximum_password_length
 from irods.logging_infrastructure import setup_rsyslog_and_logrotate, rsyslog_config_path, logrotate_config_path
 
+try:
+    import pyodbc
+except:
+    print('WARNING: pyodbc module could not be imported.', file=sys.stderr)
+    print('The pyodbc module is required for setup of provider servers.', file=sys.stderr)
```

Comment:
"provider servers" -> "catalog service providers" ?

Reply:
I like this wording. It's a little easier to read imo.